src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Extensions/JsRuntimeExtension.cs

```diff
@@ -8,5 +8,10 @@ public static async Task SetToggleBodyOverflow(this IJSRuntime jsRuntime, bool i
         {
             await jsRuntime.InvokeVoidAsync("toggleBodyOverflow", isNavOpen);
         }
+
+        public static async Task CopyToClipboardOnClickEvent(this IJSRuntime jsRuntime, string codeSampleContentForCopy)
+        {
+            await jsRuntime.InvokeVoidAsync("copyToClipboardOnClickEvent", codeSampleContentForCopy);
+        }
```

Comment:
please use a proper name for these methods both in C# and JS.
something like `CopyToClipboard`